### PR TITLE
PF-225 <adding new logic properties>

### PR DESCRIPTION
**Pull request by Anzhela-Sahakyan**

### CODE REVIEW

File: src/Form.js [hunk 1]
```diff
@@ -31,6 +32,9 @@ const Form = ({
   const isPagination = Object.keys(pagination).length !== 0
   const data = useField(updatableFields)
 
+  //this is to make Form rerender when the fields update
+  useContext(Context)
+
```
Comment (Tigran0199) [inline src/Form.js:37]: we dont need this

Comment (Anzhela-Sahakyan) [reply src/Form.js]: I removed it.

File: src/Helpers/logic.js [hunk 2]
```diff
@@ -313,18 +460,26 @@ export const actionHandler = (
...
+          updatableFields.push(`${logic[i].conditions[j].id}`)
+        } else {
+          logic[i].conditions[j].item
+            ? updatableFields.push(
+                `${logic[i].conditions[j].id}.${logic[i].conditions[j].item}`
+              )
+            : updatableFields.push(logic[i].conditions[j].id)
+        }
+      }
+    }
```
Comment (Tigran0199) [inline src/Helpers/logic.js:479]: one more case here
what if condition includes quantity and have item ?

File: src/Helpers/logic.js [hunk 3]
```diff
@@ -104,21 +193,78 @@ export const getHiddenIds = ({ logic, values, fields, formRef }) => {
...
+              String(values[key]?.key) === String(item) &&
+              values?.quantity &&
+              values?.quantity[key]
+          }
+        } else {
+          if (type === 'multipleChoice') {
+            fieldValue = {
+              value: values[key],
+              quantity: values?.quantity && values?.quantity[key]
+            }
```
[the remark this answers is not in the document]
Comment (Anzhela-Sahakyan) [reply src/Helpers/logic.js]: done)

**Pull request merged**